TASK1.3/main.py

```diff
@@ -0,0 +1,17 @@
+# Задача 6: Вы пользуетесь общественным транспортом? Вероятно, вы расплачивались за проезд и получали билет с номером.
+# Счастливым билетом называют такой билет с шестизначным номером, где сумма первых трех цифр равна сумме последних трех. 
+# Т.е. билет с номером 385916 – счастливый, т.к. 3+8+5=9+1+6. Вам требуется написать программу, которая проверяет счастливость билета.
+
+# *Пример:*
+
+# 385916 -> yes
+# 123456 -> no
+
+x = [int(x) for x in list(input())]
+
+left = sum(x[0:-3]) 
```

Comment:
Если срез берется сначала то ноль можно не указывать.
```suggestion
left = sum(x[:-3]) 
```